packages/palette/src/elements/Tags/Tags.tsx

```diff
@@ -24,28 +24,73 @@ export const Tags: React.FC<TagsProps> = ({
   displayNum = tags.length,
 }) => {
   const [expanded, setExpanded] = useState(false)
+  const [boxHeight, setBoxHeight] = useState("auto")
+  const flexContainer = useRef<HTMLDivElement>(null)
+  const animatingBox = useRef<HTMLDivElement>(null)
+  // after animating AnimatingBox height set back height to auto (to handle screen resize etc.)
+  useEffect(() => {
+    animatingBox.current.addEventListener("transitionend", e => {
+      const isAnimatingBoxTransition = Array.from(
```

Comment:
The transitions from `Tag` elements bubble up here so had to check if the `transitionend` is from the `AnimatingBox` element.

Reply:
I think you could probably do

```ts
const isAnimatingBoxTransition = e.target === animatingBox.current
```